scripts/skopeo2tree

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
```

Comment:
Maybe let's split this script into its own commit with a dedicated introduction in the commit message? :)

Reply:
I'm not sure it's worth writing a commit message right now when it's still got tons of bugs and issues.  I pushed it instead into https://github.com/ostreedev/ostree-releng-scripts/pull/14 with the other WIP/demo script.